api/updates/retrieve.js

```diff
@@ -0,0 +1,23 @@
+const router = require("express").Router();
+const { Updates } = require("../../database/models/updates");
+
+router.get("/:userID", async (req, res) => {
```

Comment:
The path shouldn't be specified here. Instead it should be mounted at this path in the API/updates/index.js file